test/MinCostFlow/test_mcf.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import shutil
+from pathlib import Path
+from src.mincostflow import MinCostFlow
+from test.MEO.test_meo import OUT_FILE
+
+TEST_DIR = 'test/MinCostFlow/'
+OUT_FILE = TEST_DIR+'output/mincostflow-output.txt'
+
+class TestMinCostFlow:
+    def test_mincostflow_required(self):
```

Comment:
Do you have the expected output file for these test cases? We could commit that as well to `test/MinCostFlow/expected/mincostflow-output.sif` even if we aren't using it yet. We may need to encode the flow and capacity in the filename so we can match it with the respective inputs.

Reply:
Yes I do
 
I also have more test graphs, so I could add all of them with their respective expected outputs